docs/matlab/Getting_Started.m

```diff
@@ -0,0 +1,250 @@
+%% Getting Started with RocketPy in MATLAB®
+% In this Live Script, you will learn how to run RocketPy using the MATLAB® 
+% environment.
+% 
+% We start by configuring the Python environment. You can do so by following 
+% the guidelines presented in the MATLAB® documentation: <https://www.mathworks.com/help/matlab/matlab_external/install-supported-python-implementation.html?searchHighlight=python&s_tid=srchtitle_python_4 
+% Configure Your System to Use Python - MATLAB® & Simulink (mathworks.com)>.
+% 
+% Once the Python environment is configured, RocketPy needs to installed using 
+% |pip| as outlined in RocketPy's documentation: <https://docs.rocketpy.org/en/latest/user/installation.html#quick-install-using-pip 
+% Installation — RocketPy documentation>.
+% 
+% Finally, all the prerequisites are complete and you can comeback to MATLAB®! 
+% We just need to set the execution mode as out of process and start working. 
+% MATLAB® can run Python scripts and functions in a separate process. Running 
+% Python in a separate process enables you to:
+%% 
+% * Use some third-party libraries in the Python code that are not compatible 
+% with MATLAB®.
+% * Isolate the MATLAB® process from crashes in the Python code.
+
+pyenv('ExecutionMode','OutOfProcess');
```

Comment:
On the first time I used Python inside Matlab, I had to run this command as:
```matlab
pyenv('ExecutionMode','OutOfProcess', 'Version', '/path/to/python/executable');
```
Because Matlab could not find the Python executable automatically, but after I ran it once, I can use this version of the command just fine. Maybe it is worth mentioning in the docs, I am not sure if I was unlucky or if it always like this.